lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -403,8 +403,9 @@ func addHopHint(hopHints *[]func(*zpay32.Invoice),
 	channel *channeldb.OpenChannel, chanPolicy *channeldb.ChannelEdgePolicy) {
 
 	hopHint := zpay32.HopHint{
-		NodeID:      channel.IdentityPub,
-		ChannelID:   channel.ShortChanID().ToUint64(),
+		NodeID: channel.IdentityPub,
+		// Zeroed out for privacy reasons.
+		// ChannelID:   channel.ShortChanID().ToUint64(),
```

Comment:
Yes, that is currently signaled implicitly by one or more zero channel ids in the route hints. It would also be nice to reduce the invoice size (and qr codes) by removing the eight zero bytes completely.